plugins/module_utils/network/asa/argspec/acls/acls.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, **kwargs):
                                         "address": {"type": "str"},
                                         "netmask": {"type": "str"},
                                         "any": {"type": "bool"},
+                                        "host": {"type": "str"},
```

Comment:
This needs to be updated in Module documentation as well or is it already updated in the module file. ?

Reply:
@rohitthakur2590 This is already part of module doc (ref: https://github.com/ansible-collections/cisco.asa/blob/master/plugins/modules/asa_acls.py#L295), facts parsing logic implementation was missed

Reply:
Now, have added merged TC around same to cover the scenario

Reply:
+1